.editorconfig

```diff
@@ -0,0 +1,170 @@
+root = true
+
+[*]
+charset = utf-8
+end_of_line = lf
+indent_size = 4
+indent_style = space
+insert_final_newline = true
+max_line_length = 99
+tab_width = 4
+trim_trailing_whitespace = true
+ij_continuation_indent_size = 8
+# ij_formatter_off_tag = @formatter:off
+# ij_formatter_on_tag = @formatter:on
+# ij_formatter_tags_enabled = false
+# ij_smart_tabs = false
```

Comment:
[question] Now were enforcing C, C++ and Python through tooling.  I'm becoming somewhat less enamoured with the army of `ij_*` entries we have here, especially unset ones.  It makes it kind of hard to grok what's actually going on, or applicable to all tools. I've been omitting them from the other new repos.  Do we think they really justify inclusion for just a single specific editor? We dont, for example list all possible generic options that we dont set, or options for other editors.

Reply:
Yeah fair enough it's quite a WOT.  I'll remove them and see if I miss them.

Reply:
Simplified significantly in 804ca886 - though I left a couple of the particularly useful ones (for me) in there.

Reply:
Yeah, the guides are def a good one 👍 thanks.